Data/ByteString/Short/Internal.hs

```diff
@@ -297,6 +424,12 @@ fromShortIO sbs = do
                         (PlainPtr mba#)
     return (BS fp len)
 
+-- | /O(1)/ Convert a 'Word8' into a 'ShortByteString'
+--
+-- @since 0.11.3.0
+singleton :: Word8 -> ShortByteString
+singleton = \w -> create 1 (\mba -> writeWord8Array mba 0 w)
+{-# INLINE singleton #-}
```

Comment:
Yes, `INLINABLE` seems like the better default, because it doesn't increase compile time as much for users.

Instead of adding a pragma on each function, we could alternatively use the [`-fexpose-all-unfoldings` flag](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/using-optimisation.html#ghc-flag--fexpose-all-unfoldings) in this module. That would reduce visual noise, and eliminate the possibility that we forget to add a pragma.